docs/b2b-edition/specs/api-v3/authentication.yaml

```diff
@@ -339,7 +339,7 @@ paths:
           description: Pagination offset
   /auth/storefront:
     get:
-      summary: Get a StoreFront OpenAPI AuthToken
+      summary: Get a Storefront OpenAPI AuthToken
```

Comment:
I feel like we should remove all the "OpenAPI" language? Because that relates to OAS, not to our platform. And we don't talk about OpenAPI like this in any of our other docs.